### ISSUE TITLE
Consider using semantic versioning

### ISSUE THREAD

**Issue by dreua** [andreafrancia/trash-cli#357]
**Is your feature request related to a problem? Please describe.**
As a downstream package co-maintainer and user I really wonder what these four numbers in a version like 0.24.5.26 mean. It is also harder to comprehend being used to the three part versions most projects use.

**Describe the solution you'd like**
Implement [semantic versioning](https://semver.org/) in the future, consider releasing 1.0.0 as this is already used in production.

**Describe alternatives you've considered**
Document what the four parts of your versioning system mean.

**Additional context**
The semver specification is geared more towards libraries with an API than user facing programs but it is still a good framework. Use your best judgement on what a breaking or "major" change is to you users. Some people also consider a change in dependencies a "major" change for user facing applications, where versions are most important to maintainers and when debugging. Bumping major can be an indicator to package maintainers that extra care or more testing should take place.

**Comment by dreua** [comment]
I realized just now that the version is based on the release date :facepalm:

**Comment by jxu** [comment]
I agree. The user interface seems to have been settled so it should be 1.0
